src/js/components/CheckBox/index.d.ts

```diff
@@ -4,6 +4,7 @@ import { A11yTitleType } from '../../utils';
 export interface CheckBoxProps {
   a11yTitle?: A11yTitleType;
   checked?: boolean;
+  defaultChecked?: boolean;
```

Comment:
If this is out of the box React input prop, it should be added automatically by specifying `JSX.IntrinsicElements['input']`. we only need the explicit addition to the type file when we are applying existing prop modification, and if that is the case, it should also be added to the TS omit functionality.

Reply:
removed